pennylane_lightning/core/src/simulators/lightning_kokkos/gates/MatrixGateFunctors.hpp

```diff
@@ -142,6 +214,51 @@ template <class PrecisionT> struct apply1QubitOpFunctor {
         arr(i1) = matrix(0B10) * v0 + matrix(0B11) * v1;
     }
 };
+
+template <class PrecisionT> struct applyNC1QubitOpFunctor {
```

Comment:
and this with [`apply1QubitOpFunctor`](https://github.com/PennyLaneAI/pennylane-lightning/blob/b491a37635145449a88e68910794179f72a6ee6d/pennylane_lightning/core/src/simulators/lightning_kokkos/gates/MatrixGateFunctors.hpp#L107) etc.